program/src/onchain.rs

```diff
@@ -473,7 +534,7 @@ mod tests {
 
             let extra_account2_info = AccountInfo::new(
                 &extra_account2_key,
-                false,
+                true,
```

Comment:
The key variable change. When making an extra transfer hook account signed, it previously caused the tests to fail.